patterns/my-hook/hooks/MyHook.template

```diff
@@ -0,0 +1,16 @@
+import React, { useState, useEffect } from 'react';
+
+function MyHook (props) {
```

Comment:
Do you mind using an arrow function ? Not very important but it would help keeping the code base consistant :-) You don't have to of course

Reply:
We don't want to use an arrow function here. Arrow functions are anonymous functions, that means they are re-evaluated in memory at each file's call. here we want our hook's source code to be evaluated only once, when the hook is called for the first time. That's why it's better to use `function()` syntax if we want to optimize the library.

Please feel free to correct me if I'm wrong.

Reply:
Alright, I couldn't find evidence backing up what you just said... Would you mind providing links to websites/blogs discussing that ? I thought that storing the arrow function in a variable made it like a normal function, except that it contains no context, so `this` refered to its parent...

Reply:
Actually that's something I learnt through some projects. But here since we export the named arrow function I think that my previous statement doesn't apply. So I'll change it to an arrow function :)